pkg/products/marin3r/reconciler.go

```diff
@@ -147,24 +147,11 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, nil
 	}
 
-	// Ensures that for an upgrade scenario we are removing V2 EnvoyConfigRevision before
-	// initializing an upgrade to marin3r 0.10.0
-	// TODO remove after the next release from here
-	listOptions := []k8sclient.ListOption{
-		k8sclient.MatchingLabels(map[string]string{
-			"marin3r.3scale.net/envoy-api": "v2",
-		}),
-		k8sclient.InNamespace(fmt.Sprintf("%s3scale", installation.Spec.NamespacePrefix)),
-	}
-	envoyConfigRevisions := &v1alpha1.EnvoyConfigRevisionList{}
-	_ = client.List(ctx, envoyConfigRevisions, listOptions...)
-	for _, envoyConfigRevision := range envoyConfigRevisions.Items {
-		err = client.Delete(ctx, &envoyConfigRevision)
-		if err != nil || !k8serr.IsNotFound(err) {
-			return phase, err
-		}
+	phase, err = reconcileEnvoyConfigRevisionsDeletion(ctx, client, *r.installation)
```

Comment:
```suggestion
	phase, err = reconcileEnvoyConfigRevisionsDeletion(ctx, client, installation)
```
we usually pass the in memory copy of installation down in the functions rather than the one belonging to the reconciler instantiation.

Reply:
Hey, thanks, good point, actually, looking at the code, going to change it to string rather than entire rhmi.